freqtrade/freqtradebot.py

```diff
@@ -443,6 +451,53 @@ def create_trade(self, pair: str) -> bool:
         else:
             return False
 
+#
+# BUY / increase positions / DCA logic and methods
+#
+    def process_open_trade_positions(self):
+        """
+        Tries to execute additional buy or sell orders for open trades (positions)
+        """
+        # Walk through each pair and check if it needs changes
+        for trade in Trade.get_open_trades():
+            try:
+                self.adjust_trade_position(trade)
+            except DependencyException as exception:
+                logger.warning('Unable to adjust position of trade for %s: %s',
+                               trade.pair, exception)
+
+    def adjust_trade_position(self, trade: Trade):
```

Comment:
Please change the name of this function to "something" else.
Having the name identical to the strategy callback is confusing - and can potentially lead to odd, accidental bugs.

Reply:
That name has been changed to `check_and_call_adjust_trade_position`